third_party/3/jwt/algorithms.pyi

```diff
@@ -0,0 +1,2 @@
+class Algorithm:
```

Comment:
Oh, I didn't actually realize that was different from `class Algorithm: ...`.  I guess what I have currently means that `Algorithm` is a class without any methods? I'll set it to `Any`.